query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/update/UpdateStart.java

```diff
@@ -41,4 +41,24 @@
    */
   @NonNull
   UpdateStart usingTimestamp(@NonNull BindMarker bindMarker);
+
+  /**
+   * Adds a USING TTL clause to this statement with a literal value. Setting a value of {@code 0}
+   * will clear a previously set TTL.
+   *
+   * <p>If this method or {@link #usingTtl(BindMarker) } is called multiple times, the value from
+   * the last invocation is used.
+   */
+  @NonNull
+  UpdateStart usingTtl(int ttl);
+
+  /**
+   * Adds a USING TTL clause to this statement with a bind marker. Setting a value of {@code 0} will
+   * clear a previously set TTL.
+   *
+   * <p>If this method or {@link #usingTtl(int) } is called multiple times, the value from the last
+   * invocation is used.
+   */
+  @NonNull
+  UpdateStart usingTtl(@NonNull BindMarker bindMarker);
```

Comment:
I see that usingTtl methods are repeated in multiple interfaces. Maybe we should extract them to interface `ttlSupport` or smth similar and just implements that in all interfaces that need ttl support? (to not duplicate the code)

Reply:
I would, but both `Insert` and `UpdateStart` inherit from different super interfaces already, and neither of those should have `usingTtl` added to them.

Reply:
Just for two methods that are repeated two times I think it's not worth it.